tmt/utils.py

```diff
@@ -725,7 +748,7 @@ def run(self,
 
         try:
             return self._run(
-                command, cwd, shell, env, log, join, interactive, timeout)
+                command, cwd, shell, env, log if not silent else None, join, interactive, timeout)
```

Comment:
@psss Is this a good way how to suppress the actual logging?

Reply:
I'm not sure. The condition is definitely correct.

I would suspect that with `silent=True`, it wouldn't make sense to even specify `log` parameter on the caller's side, because, well, they wanted silence... Maybe `silent is True and log is not None` should be handled as an incompatible combination of inputs, with an `assert` or exception. Probably not a crucial thing to worry about... @psss what do you think?

Reply:
>  Maybe `silent is True and log is not None` should be handled as an incompatible combination of inputs, with an `assert` or exception. 

Even when `silent = True` one may pass a `message` that would be printed to a log.

Reply:
Right, let's not dissect it more, not that important.